libmodernize/main.py

```diff
@@ -145,3 +166,12 @@ def main(args=None):
 
     # Return error status (0 if rt.errors is zero)
     return int(bool(rt.errors))
+
+def fix_line_endings(linesep):
+    if sys.version_info < (3, 0):
+        os.linesep = linesep
+    else:
+        def _to_system_newlines(s):
+            return s.replace(os.linesep, linesep)
+
+        refactor._to_system_newlines = _to_system_newlines
```

Comment:
An alternative approach is to do the ``refactor._to_system_newlines`` patch on Python 2 as well; I think that will work.